### PR TITLE
Add patches and configuration

### PR DESCRIPTION
**Pull request by edouardb**

### CODE REVIEW

File: patches/etc/update-motd.d/70-gogs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+PUBLIC_IP=$(oc-metadata --cached PUBLIC_IP_ADDRESS)
+
+cat <<EOF
+Hi Cloud Rider,
+Welcome to the Gogs image!
+Initialize Gogs at http://${PUBLIC_IP}.
+MySQL is configured for user root with password {mysql_root_password}.
+Gogs database name is gogs.
```
Comment (aimxhaisse) [inline patches/etc/update-motd.d/70-gogs]: Gogs' database name is gogs.

File: patches/root/README [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Hi Cloud Rider,
+Welcome to the Gogs image!
+
+Initialize Gogs at http://${PUBLIC_IP}.
+
+MySQL is configured for user root with password {mysql_root_password}.
+Gogs database name is gogs.
```
Comment (aimxhaisse) [inline patches/root/README]: Same

**Comment by moul** [comment]
LGTM

File: patches/etc/init/init-gogs.conf [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# init etherpad configuration
+
+description "initializes etherpad on first start"
```
Comment (moul) [inline patches/etc/init/init-gogs.conf]: etherpad -> gogs

**Pull request merged**